synapse/storage/databases/main/room_batch.py

```diff
@@ -36,3 +36,16 @@ async def get_insertion_event_by_batch_id(
             retcol="event_id",
             allow_none=True,
         )
+
+    async def store_state_group_id_for_event_id(
+        self, event_id: str, state_group_id: int
+    ) -> Optional[str]:
+        {
```

Comment:
Not sure what's going on here. The braces to enclose the `await` is legit syntax (it'll build a set, like `{"hello"}`). As written the function is going to return `None`.

Reply:
See #11310